ding/rl_utils/ppo.py

```diff
@@ -180,7 +213,16 @@ def ppo_policy_error(
         approx_kl = (logp_old - logp_new).mean().item()
         clipped = ratio.gt(1 + clip_ratio) | ratio.lt(1 - clip_ratio)
         clipfrac = torch.as_tensor(clipped).float().mean().item()
-    return ppo_policy_loss(policy_loss, entropy_loss), ppo_info(approx_kl, clipfrac)
+
+    if logit_pretrained is not None:
+        dist_pretrained = torch.distributions.categorical.Categorical(logits=logit_pretrained)
+        logp_pretrained = dist_pretrained.log_prob(action)
+        logr = logp_new - logp_pretrained
+        kl_div = calculate_kl_div(logr, kl_type)
+    else:
+        kl_div = 0
+
+    return ppo_policy_loss(policy_loss, entropy_loss), ppo_info(approx_kl, clipfrac, kl_div)
```

Comment:
`kl_div` should be one of the output item of `ppo_policy_loss`, `ppo_info` is just the monitor variable